app/build.gradle

```diff
@@ -141,12 +142,16 @@ dependencies {
     implementation "androidx.compose.animation:animation:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
     implementation "androidx.constraintlayout:constraintlayout-compose:1.0.0"
-    implementation 'androidx.navigation:navigation-compose:2.5.0-beta01'
+    implementation 'androidx.navigation:navigation-compose:2.5.0-rc01'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-compose:2.4.1'
     implementation 'androidx.hilt:hilt-navigation-compose:1.0.0'
     implementation 'io.coil-kt:coil-compose:2.0.0-rc03'
     implementation 'com.airbnb.android:lottie-compose:5.0.3'
 
+    //Spotify
+    implementation 'com.spotify.android:auth:1.2.5'
```

Comment:
Last time I checked I was unable to find the source of Spotify Android libraries. Closed source libraries are not allowed in apps to be published on F-Droid. Can you please confirm and share the relevant source of the Spotify libs in this PR description for record?

Reply:
This dependency has been removed and we only require https://github.com/spotify/android-sdk/releases/tag/v0.7.2-appremote_v1.2.3-auth the aar file here which is open-sourced and available.

Reply:
We have decided to have different build variants to tackle this situation. Flavours will be added in later stages to distinguish the F-Droid version.